lib/easypost/util.rb

```diff
@@ -151,15 +158,17 @@ def self.validate_webhook(event_body, headers, webhook_secret)
     easypost_hmac_signature = headers['X-Hmac-Signature']
 
     if easypost_hmac_signature.nil?
-      raise EasyPost::Error.new('Webhook received does not contain an HMAC signature.')
+      msg = EasyPost::Constants::WEBHOOK_MISSING_SIGNATURE
+      raise EasyPost::Errors::SignatureVerificationError.new(msg)
```

Comment:
suggestion: inline (you see the pattern now right? There are a few more of these and I'll stop marking them here)